agent/src/IngestDetailsRepository.php

```diff
@@ -128,7 +128,7 @@ private function parseResponse(ResponseInterface $response): IngestDetails
             ! is_int($data['refresh_in'] ?? null) ||
             ! is_string($data['ingest_url'] ?? null)
         ) {
-            throw new RuntimeException("Invalid authentication response [{$body}].");
+            throw new RuntimeException("Invalid authentication response [{$body}]");
```

Comment:
Removing the trailing periods from all the exception message to have consistent log messages.